src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -126,6 +127,9 @@ const webhookRouter = async (context: WebhookContext) => {
 		case "code_scanning_alert":
 			await GithubWebhookMiddleware(codeScanningAlertWebhookHandler)(context);
 			break;
+		case "dependabot_alert":
+			await GithubWebhookMiddleware(dependabotAlertWebhookHandler)(context);
```

Comment:
We should feature flag this webhook handler. This is our FF in launch darkly https://app.launchdarkly.com/github-for-jira/production/features/enable-github-security-in-jira/targeting and this is the flag key I am using in my PR 
https://github.com/atlassian/github-for-jira/pull/2222/files#diff-0bbb4cf9c271e6064c5575a9375ac6b7503b922c7fab9f49fb185d77914edbddR31

Reply:
Sorry just saw your comment in the description. I guess up to @atrigueiro and @rachellerathbone if you are ok with us adding this FF later?

Reply:
nws, ill just add it.